### ISSUE TITLE
awscfncli2/config/schema_v2.json is missing in pip package

### ISSUE THREAD

**Issue by ouelletandrebni** [Kotaimen/awscfncli#35]
Hi,

Steps to re-create the issue:

1. Install awscfncli2 by running ```pip install --upgrade --user awscfncli2```
2. Run the following cfn-cli command: ```cfn-cli --profile myawsprofile -f ./cfnconfig.yml stack deploy```
3. The following error is received:
```
FileNotFoundError: [Errno 2] No such file or directory: '/home/user/.local/lib/python3.5/site-packages/awscfncli2/config/schema_v2.json'
```

Upon further investigation,  the files ```awscfncli2/config/schema_v1.json``` and ```awscfncli2/config/schema_v2.json``` are in the master branch in github for this project, but they are missing in in pypi (at least they are missing when I download the 2.0.0.tar.gz from [here](https://pypi.org/project/awscfncli2/2.0.0/#files) and check inside)

I've attached the full traceback in the traceback.txt file.
[traceback.txt](https://github.com/Kotaimen/awscfncli/files/2291388/traceback.txt)

**Comment by ouelletandrebni** [comment]
Also, here is the contents of my cfnconfig.yml file with values changed to generic.

```
Version: 2
Stages:
  Default:
    Stack:
      TemplateBody: ../../dist/cfnstack-packaged.yml
      Capabilities: [CAPABILITY_IAM, CAPABILITY_NAMED_IAM]
      Region:       us-east-1
      StackName:    mystackname
      Parameters:
        DomainNamePublic: mydomain.com
        DomainNamePrivate: mydomain.int
        BusinessUnit: mybusinessunit
        Project: myproject
        ProjectFullName: MyProjectFullName
        ProjectKey: myprojectkey
        Component: mycomponent
        ComponentSuffix: component
        AWSAccountName: myawsaccount
        EnvInfra: myenvname
        EnvType: sbx
      Tags:
        BusinessUnit:       mybusinessunit
        Project:            myproject
        Component:          mycomponent
        Environment:        myenvname
        Creator:            ouellet.andre@brunswicknews.com
```

**Comment by GlieseRay** [comment]
Hi, @ouelletandrebni 
Sorry for bring the trouble. This is a bug in the packaging code. We have fixed #36 and deployed a new version 2.0.1. Please update to the latest version to see if it fix you issues. Thanks.

**Comment by ouelletandrebni** [comment]
Hi,

It's working great now.

I ran the following pip upgrade command, and this got me to version 2.0.1

    #$ *pip install --upgrade --user awscfncli2*
    ...
    Successfully installed awscfncli2-2.0.1

I can also confirm that the missing files are now in place

    #$ ls -la
/home/user/.local/lib/python3.5/site-packages/awscfncli2/config/
    total 92
    drwxrwxr-x 3 user user  4096 Aug 16 15:34 .
    drwxrwxr-x 5 user user  4096 Aug 16 15:34 ..
    -rw-rw-r-- 1 user user 10328 Aug 16 15:34 config.py
    -rw-rw-r-- 1 user user    68 Aug 16 15:34 exceptions.py
    -rw-rw-r-- 1 user user   118 Aug 16 15:34 __init__.py
    drwxrwxr-x 2 user user  4096 Aug 16 15:34 __pycache__
    -rw-rw-r-- 1 user user   577 Aug 16 15:34 schema.py
*    -rw-rw-r-- 1 user user   162 Aug 16 15:34 schema_v1.json*
*    -rw-rw-r-- 1 user user  2817 Aug 16 15:34 schema_v2.json*

Thanks for the quick turn around on fixing this :)


*Andre Ouellet*
Senior Systems Analyst | Brunswick News Inc.
------------------------------

Mobile: 506-380-0291
ouellet.andre@brunswicknews.com
https://www.telegraphjournal.com/
------------------------------
